esp-hal/src/spi/master.rs

```diff
@@ -3379,6 +3401,23 @@ impl Driver {
         future.await;
     }
 
+    fn disable_half_duplex(&self) -> Result<(), Error> {
```

Comment:
I suppose this is just `setup_full_duplex` at the end of the day.
Hopefully this can be unified with `setup_half_duplex` in future.

Reply:
sure - probably setup_full_duplex sounds better 👍